internal/pipeline/examples/simplepackage/configs/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: kptfile
```

Comment:
This should be name of the package. `simplepackage` in this case.

While typing this I realized why not call this package something useful such as `app` or something.

Reply:
done